pkg/specgenutil/specgen.go

```diff
@@ -1297,3 +1301,293 @@ func GetResources(s *specgen.SpecGenerator, c *entities.ContainerCreateOptions)
 	}
 	return s.ResourceLimits, nil
 }
+
+func parseUint(val string) (uint, error) {
+	valUint, err := strconv.ParseUint(val, 10, 64)
+	if err != nil {
+		return 0, err
+	}
+	return uint(valUint), nil
+}
+
+func GetChangedHealthCheckConfiguration(cmd *cobra.Command) define.UpdateHealthCheckConfig {
```

Comment:
cobra imports should not be in specgen, cli parts should all stay under cmd/podman.
I See noe reason why the function is here.